seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -180,7 +180,8 @@
         "<code>&thinsp;&thinsp;return(val+1);</code>",
         "<code>});</code>",
         "",
-        "The map method is one of the easiest ways to iterate through an array or object there is. Let's use it now."
+        "The map method is one of the easiest ways to iterate through an array or object there is. Let's use it now.",
+        "Use the map function to add 3 to every value in the variable `array`"
```

Comment:
Thanks! I think you'll need to use `<code></code>` tags here instead of backticks, though.